internal/command/bind.go

```diff
@@ -175,7 +91,7 @@ func NewBindCommand(p *KameletPluginParams) *cobra.Command {
 				if err != nil {
 					return err
 				}
-				return printer.PrintObj(&binding, out)
+				return printer.PrintObj(binding, out)
```

Comment:
Why would you do that on a create ? Might be helpful for a "describe" but not so much for 'create'.

Also, I get this error if I try to specify `-o yaml`:

```
kn source kamelet bind chuck-norris-source --sink service:edisplay -o yaml
kamelet binding "chuck-norris-source-to-service-edisplay" updated
Error: missing apiVersion or kind; try GetObjectKind().SetGroupVersionKind() if you know the type
```

Reply:
it is for printing the created binding resource for debug reasons.

the error you get is related to missing GroupVersionKind on unstructured objects in json printer on kn-client and a fix is provided in https://github.com/knative/client/pull/1278

Reply:
Interesting, because in the client itself we never experienced that issue (as you see from the comments over there). So now that we have a produce let me debug it from this plugin.